zschema/leaves.py

```diff
@@ -411,18 +415,42 @@ class IndexedBinary(Binary):
 class DateTime(Leaf):
 
     ES_TYPE = "date"
-    BQ_TYPE = "TIMESTAMP"
+    BQ_TYPE = "DATETIME"
     EXPECTED_CLASS = [str, int, unicode]
 
     VALID = "Wed Jul  8 08:52:01 EDT 2015"
     INVALID = "Wed DNE  35 08:52:01 EDT 2015"
 
+    MIN_VALUE = "1752-01-01 00:00:00.000000"
+    MAX_VALUE = "9999-12-31 00:00:00.000000"
+
+    def __init__(self, *args, **kwargs):
+        super(DateTime, self).__init__(*args, **kwargs)
+
+        if self.min_value:
+            self._min_value_dt = dateutil.parser.parse(self.min_value, ignoretz=True)
+        else:
+            self._min_value_dt = dateutil.parser.parse(self.MIN_VALUE, ignoretz=True)
+
+        if self.max_value:
+            self._max_value_dt = dateutil.parser.parse(self.max_value, ignoretz=True)
+        else:
+            self._max_value_dt = dateutil.parser.parse(self.MAX_VALUE, ignoretz=True)
```

Comment:
Are we sure we want `ignoretz` vs explicitly marking this as being UTC?

Reply:
It's unclear whether the incoming timestamp will have a specified timezone, and will throw an error if they don't, because we'll be comparing TZed and non-TZed timestamps.

The only time this matters is when someone puts in year 9999, which shouldn't ever be happening.

Reply:
You can't compare TZ'ed and non-TZ'ed timestamps?

Sigh.